src/components/addBookForm.js

```diff
@@ -1,20 +1,42 @@
-/* eslint-disable react/jsx-props-no-spreading, no-console */
-import React from 'react';
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
- [ ] Please `Linters` are there to help us stick to best practices, instead of disabling `eslint`, it will be nice to fix the issue.

Reply:
I'm not able to solve the issue about that rule, the syntax for the useForm import requires the spreading at the props.
If you have an idea I'd like to try it.